schemas/Screen.json

```diff
@@ -18,7 +18,8 @@
         {"const": "RoomSettings", "description": "The settings screen shown from the Room Details screen."},
         {"const": "RoomNotifications", "description": "The notifications settings screen shown from the Room Details screen."},
         {"const": "RoomDirectory", "description": "The screen that lists public rooms for you to discover."},
-        {"const": "StartChat", "description": "The screen shown to create a new (non-direct) room."},
+        {"const": "RoomFilter", "description": "The screen that lists all the user's rooms and let them filter the rooms."},
```

Comment:
I'm pretty sure that this isn't the same view as `MobileSearchRooms` (which may be iOS only anyway) but pointing it out in case it is :)

I only ask as on iOS the rooms are filtered from the `Home`/`MobileFavourites`/`MobilePeople`/`MobileRooms` tabs so (I don't imagine) we wouldn't track this as a screen there.

Reply:
Yes, on Android, rooms can be filtered using a dedicated screen (it will probably change in the future, IA is coming)